packages/sanity/src/core/store/bundles/types.ts

```diff
@@ -8,6 +8,9 @@ import {type PartialExcept} from '../../util'
 import {type MetadataWrapper} from './createBundlesMetadataAggregator'
 import {type bundlesReducerAction, type bundlesReducerState} from './reducer'
 
+/** @internal */
+export type releaseType = 'asap' | 'scheduled' | 'undecided'
+
```

Comment:
will disappear once https://github.com/sanity-io/sanity/pull/7619/files is merged